test/parallel/test-cli-node-print-help.js

```diff
@@ -44,11 +44,13 @@ function validateNodePrintHelp() {
 function testForSubstring(options) {
```

Comment:
could we add:
```
{ compileConstant: true,
  flags: [
    '-',
    '--',
    '--abort-on-uncaught-exception',
    '-c',
    '--check',
    '-e',
    '--eval script',
    '--experimental-modules',
    '--experimental-repl-await',
    '--experimental-vm-modules',
    '-h',
    '--help',
    '-i',
    '--interactive',
    '--napi-modules',
    '--no-deprecation',
    '--no-force-async-hooks-checks',
    '--no-warnings',
    '-p',
    '--pending-deprecation',
    '--preserve-symlinks',
    '--preserve-symlinks-main',
    '--print',
    '--prof',
    '--prof-process',
    '-r',
    '--redirect-warnings=file',
    '--require',
    '--throw-deprecation',
    '--trace-deprecation',
    '--trace-event-categories',
    '--trace-event-file-pattern',
    '--trace-events-enabled',
    '--trace-sync-io',
    '--trace-warnings',
    '--track-heap-objects',
    '-v',
    '--v8-options',
    '--v8-pool-size=num',
    '--version',
    '--zero-fill-buffers',
    'NODE_DEBUG',
    'NODE_DEBUG_NATIVE',
    'NODE_DISABLE_COLORS',
    'NODE_EXTRA_CA_CERTS',
    'NODE_NO_WARNINGS',
    'NODE_OPTIONS',
    'NODE_PATH',
    'NODE_PENDING_DEPRECATION',
    'NODE_PRESERVE_SYMLINKS',
    'NODE_REDIRECT_WARNINGS',
    'NODE_REPL_HISTORY',
    'OPENSSL_CONF',
  ]
}
```

Reply:
If you submit a PR with that I’ll approve it, yes.